src/components/TextField/TextField.tsx

```diff
@@ -181,10 +181,10 @@ export const TextField = ({
       )}
 
       <div className={styles['text-field__body']}>
-        <TextInput
+        <InputField
           aria-describedby={ariaDescribedByVar}
           aria-invalid={!!isError}
-          data-bootstrap-override="textinput"
+          data-bootstrap-override="InputField"
```

Comment:
I'm just wondering what the capitalization pattern is we want to use for these because the other `data-bootstrap-override` attribute values are all lowercase.

Reply:
Nice catch, will just say `inputfield`